services/uploads/serverless.yml

```diff
@@ -86,6 +86,10 @@ resources:
       Type: AWS::S3::Bucket
       Properties:
         BucketName: !Sub ${self:service.name}-${self:custom.stage}-attachments-${AWS::AccountId}
+        BucketEncryption:
```

Comment:
@mdial89f @ayoneeyee 
dumb question... when we turn on encryption I assume it just encrypts everything that was already in the bucket already and move along as if nothing changed from a functional standpoint

Reply:
No, that's a good question.  I don't have the answer... @ayoneeyee  ?

Reply:
Only new objects will be encrypted. Existing objects will not be encrypted. So we need to come up with a solution for encrypting all existing objects.

Reply:
@ayoneeyee The documentation above is good.  I'm approving.

You might consider putting a summary of steps to take after merge deployment.  I think the summary is basically that they need to run that one command for each S3 bucket "aws s3 cp s3://attachmentsbucket/ s3://attachmentsbucket/ --sse AES256 --recursive" and "aws s3 cp s3://clamdefsbucket/ s3://clamdefsbucket/ --sse AES256 --recursive"

This can be changed anytime if we want, so I'm going to approve since this is complete.